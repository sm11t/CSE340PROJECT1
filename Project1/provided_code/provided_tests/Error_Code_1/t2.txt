TASKS
    1 2 3
POLY
    F = x + 2;
    F(Y,Z) = 3;
    G = x - 1;
    G
    (Y,
    Z) = Y + 3 (Y - 2)^2 + Z;
    F1 = x - 2;
    F = x + 2 + 2;
EXECUTE
    INPUT A;
    w = F(A);
    OUTPUT w;
INPUTS
    3 4 5