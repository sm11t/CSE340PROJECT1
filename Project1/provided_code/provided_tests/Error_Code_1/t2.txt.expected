Semantic Error Code 1: 5 7 11 
